tests/flow/test_constraint.py

```
@@ -0,0 +1,1061 @@
+            self.env.assertTrue(False)
+        except ResponseError as e:
+            self.env.assertContains("mandatory constraint violation: node with label Person missing property height", str(e))
+
+        #-----------------------------------------------------------------------
+        # create a node that violates the unique constraint on point data ignored
+        #-----------------------------------------------------------------------
+
+        try:
+            g.query("MATCH (p:Person) CREATE (n:Person{height:p.height + 1, loc: p.loc}) DELETE n")
+            self.env.assertTrue(True)
+        except ResponseError as e:
```

Ruff (0.12.2)

`ResponseError` may be undefined, or defined from star imports

(F405)